driver-adapters/postgresql-cf-basic/prepare.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+export PRISMA_TELEMETRY_INFORMATION='ecosystem-tests driver-adapters pg-cf-basic build'
```

Comment:
Maybe rename `pg-cf-basic` everywhere to `postgresql-cf-basic` including file names (or the opposite)